sentences/nl/_common.yaml

```diff
@@ -49,13 +49,14 @@ lists:
       - in: "f"
         out: "fahrenheit"
 expansion_rules:
+  lamp: "(lamp  | lampen | lichten | verlichting )"
```

Comment:
```suggestion
  lamp: "(lamp  | lampen | lichten | verlichting)"
```

Reply:
Ik zou ook de/het hier toevoegen als optioneel om op te matchen

Reply:
ok, wist niet zeker of er ook gematched moet worden op "incorrecte" gramatica. 
Aangepast :)